_data/master/navbars/usage.yml

```diff
@@ -55,6 +55,8 @@ toc:
   path: /usage/configuration/conntrack
 - title: Calico for OpenStack
   section:
+  - title: Endpoint Labels
```

Comment:
Are there any good places to link to this from to make it more discoverable?

Reply:
We could contextualize this better if we said a lot more about when and how SGs and Calico Policy can be interleaved, what namespace OpenStack resources are mapped into etc.  But that would take more work, and I wasn't sure if it was justified, bearing in mind that the folks who are interested in this detail already know why it's useful to them.

(That is a pretty strange attitude to take to doc, I admit....)

Reply:
We have been giving some thought to the overall site organization and this in particular is a section that could benefit from relocation, however, for now, I think it is probably fine.